public/components/trace_analytics/components/traces/trace_view.tsx

```diff
@@ -250,6 +256,37 @@ export function TraceView(props: TraceViewProps) {
     );
   };
 
+  const TracetoLogsButton = () => {
+    return useMemo(
```

Comment:
useMemo should be called statically, maybe remove the outer function wrap or move useMemo to outside?

Reply:
updated [3de427e](https://github.com/opensearch-project/dashboards-observability/pull/2398/commits/3de427e9fe08191d804ca0496fec4e718e398b32)